packages/models-library/src/models_library/api_schemas_storage/storage_schemas.py

```diff
@@ -370,3 +377,53 @@ def ensure_consistent_entries(self: Self) -> Self:
 
 class SoftCopyBody(BaseModel):
     link_id: SimcoreS3FileID
+
+
+class PathMetaDataGet(BaseModel):
+    path: Annotated[Path, Field(description="the path to the current path")]
+    display_path: Annotated[
+        Path, Field(description="the path to display with UUID replaced")
+    ]
+
+    file_meta_data: Annotated[
+        FileMetaDataGet | None,
+        Field(description="if filled, this is the file meta data of the s3 object"),
+    ] = None
+
+    model_config = ConfigDict(
+        extra="forbid",
+        json_schema_extra={
```

Comment:
MINOR: I start using callable instead of constants for `json_schema_extra`

some reason:
- if not not called, then it is not loaded in memory
   - loaded for auto test examples
   - partially loaded for openapi specs (note that `exampleS` is not included)
- avoid static linter error hell


See e.g.
https://github.com/itisfoundation/osparc-simcore/blob/a9c4d5450d916851086772705ba9d6c8883b0240/packages/models-library/src/models_library/api_schemas_webserver/groups.py#L137

Reply:
oh nice. I was wondering where to look for that stuff. cool, will change